README.md

```diff
@@ -241,6 +244,36 @@ Add the following to your composer.json file:
       }
     }
   },
+  {
+    "type": "package",
+    "package": {
+      "name": "junaidbhura/wpml-sitepress-multilingual-cms",
```

Comment:
@mcaskill Does it make sense to shorten this? What is the name of the plugin that gets downloaded?

Reply:
The ZIP from wpml.org uses the directory name `sitepress-multilingual-cms`.